langchain/src/vectorstores/tests/vectara.int.test.ts

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable no-process-env */
+import { test, expect, beforeAll } from "@jest/globals";
+import { FakeEmbeddings } from "../../embeddings/fake.js";
+import { OpenAIEmbeddings } from "../../embeddings/openai.js";
+import { Document } from "../../document.js";
+import { VectaraLibArgs, VectaraStore } from "../vectara.js";
+
+describe.skip("VectaraStore", () => {
+  describe("fromTexts", () => {
+    const args: VectaraLibArgs = {
+      customer_id: Number(process.env.VECTARA_CUSTOMER_ID) || 0,
+      corpus_id: Number(process.env.VECTARA_CORPUS_ID) || 0,
+      api_key: process.env.VECTARA_API_KEY || "",
+    };
+
+    test("with embeddings", () => {
+      expect(() =>
+        VectaraStore.fromTexts([], [], new OpenAIEmbeddings(), args)
+      ).toThrow(
+        "Vectara uses its own embeddings, so you don't have to provide any. Provide an instance of FakeEmbeddings to VectaraStore.fromTexts, instead of OpenAIEmbeddings."
```

Comment:
@ofermend @aamirbutt How do you feel about this behavior? If you think it's helpful I'll also add guards against the use of `embeddings` in other methods, and do something similar for `k`.

Reply:
I like this. Very useful for the user. 
What do you mean by "doing the same for k" - I think k should be a valid variable we use - does it not provide the numResults? (see earlier comment)

Reply:
I asked @aamirbutt about `k` and it sounded like it's something that's already abstracted by Vectara? Honestly, I'm not familiar enough with this domain to understand whether `k` and `numResults` are the same thing. Based on how `k` is being used in other tests, it looks like the same thing to me but I don't know. I'll leave it to you and Aamir to work out.